webpack.config.ts

```diff
@@ -0,0 +1,75 @@
+import * as autoprefixer from "autoprefixer";
+import * as path from "path";
+import * as webpack from "webpack";
+import "webpack-dev-server"; // Side-effect import just to get devServer types.
+
+const config: webpack.Configuration = {
+  entry: "./docs/docs.jsx",
+  output: {
+    path: path.resolve(__dirname, "docs"),
+    filename: "bundle.js",
+    publicPath: "/",
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".jsx", ".less"],
+    alias: {
+      src: path.resolve(__dirname, "src"),
+      utils: path.resolve(__dirname, "src/utils"),
+      less: path.resolve(__dirname, "src/less"),
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(eot|otf|woff|svg|ttf|png|jpg)$/,
+        loader: "url-loader",
+      },
+      {
+        test: /\.(less|css)$/,
+        use: [
+          "style-loader",
+          "css-loader",
+          {
+            loader: "postcss-loader",
+            options: {
+              postcssOptions: {
+                plugins: [autoprefixer],
+              },
+            },
+          },
+          "less-loader",
+        ],
+      },
+      {
+        test: /\.tsx?$/,
+        loader: "awesome-typescript-loader",
+        options: {
+          configFileName: "./tsconfig.docs.json",
+          ignoreDiagnostics: [2307],
+        },
+      },
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: "babel-loader",
+      },
+    ],
+  },
+  optimization: {
+    nodeEnv: process.env.NODE_ENV || "development",
+  },
+  devServer: {
+    host: "0.0.0.0",
+    port: 5010,
+    contentBase: "docs/",
+    inline: true,
+    watchContentBase: true,
+    hot: true,
```

Comment:
this line is new, compared to the previous config - enabled hot reloading to make rebuilds a little faster